src/ResourceManager/Intune/Commands.Intune/Commands.Intune.Test.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@rnikhilkumar Please follow the pattern for cmdlets - you should have a directory for yout cmdlets: Intune/Commands.Intune/  and a directory for your tests:  Intune/Commands.Intune.Test/